clients/src/main/java/org/apache/kafka/common/utils/Java.java

```diff
@@ -44,6 +44,10 @@ public static boolean isIbmJdk() {
         return System.getProperty("java.vendor").contains("IBM");
     }
 
+    public static boolean isIbmJdkSemeru() {
```

Comment:
Instead of having this with the logic in various places, can we not stream IBM Semeru is plain openjdk? If so, we could change the check for IBM JDK to exclude IBM Semeru.

Reply:
I don't know if we can treat the IBM Semeru JDK as openjdk in all places where `isIbmJdk()` is being used. The changes here fix the places where the IBM security classes were being incorrectly specified for Semeru. I could look at the other places but since there aren't tests that are failing or obvious regression, I would need to investigate more.